nbsite/nbbuild.py

```diff
@@ -326,6 +329,9 @@ def evaluate_notebook(nb_path, dest_path=None, skip_exceptions=False,substring=N
     if skip_output:
         preprocessors.append(SkipOutput(skip_output))
     ret = nb_to_html(dest_path, preprocessors=preprocessors)
+
+    if not no_interactivity_warning:
+        ret += '<div id="scroller-right">Not all interactivity in this notebook may work.</div>'
```

Comment:
Maybe: 

This web page was generated from a Jupyter notebook and not all interactivity will work on this website. _Download_and_run_ or _Launch_on_Binder_ for full Python-backed interactivity.